cmd/neblictl/internal/controlplane/commands.go

```diff
@@ -696,6 +696,12 @@ func NewCommands(controlPlaneExecutors *Executors, controlPlaneCompleters *Compl
 						Optional:    true,
 						Default:     "10",
 					},
+					{
+						Name:        "export-template",
```

Comment:
As it's expecting a CEL expression. Would not be better to call that `export-expression` for the sake of consistency?

Reply:
well, now is using CEL but that is an implementation detail so I consider it a template.